pandas/tests/extension/test_numpy.py

```diff
@@ -415,10 +415,6 @@ def test_setitem_mask(self, data, mask, box_in_series):
     def test_setitem_mask_raises(self, data, box_in_series):
         super().test_setitem_mask_raises(data, box_in_series)
 
-    @skip_nested
```

Comment:
do we have a setitem test?

Reply:
Yes, I added a case containing NA to the the `test_setitem_mask` test Joris had merged previously (should be line 101 of `setitem.py` above)